### PR TITLE
Update CVRPTW generator scaling

### LINKED ISSUE #258: [BUG] plotting out of scale
## Describe the bug

CVRPTW generator scales both time and location coordinates with the same factor (max_time). This notably results in a bad rendering.

## To Reproduce

Initialize the CVRPTWGenerator with specific parameters, ensuring that scale=True and max_time is set to a particular value.​
Generate a set of instances using this generator.​
Examine the spatial distribution of the generated locations.

```python
from rl4co.envs.routing.cvrptw.env import CVRPTWEnv
env = CVRPTWEnv(generator_params={'num_loc':20,'scale':True})
env.render(td_init[0],actions=torch.tensor([0,1]))
```

## System info

rl4co : 0.5.2

## Additional context

The render function could also be updated (the axs limit currently assume you have scaled the problem)

## Reason and Possible fixes

Change this : 

if self.scale:
    durations = durations / self.max_time
    min_times = min_times / self.max_time
    max_times = max_times / self.max_time
    td["depot"] = td["depot"] / self.max_time
    td["locs"] = td["locs"] / self.max_time

To this : 
if self.scale:
    durations = durations / self.max_time
    min_times = min_times / self.max_time
    max_times = max_times / self.max_time
    td["depot"] = td["depot"] / self.max_loc
    td["locs"] = td["locs"] / self.max_loc

## Checklist

- [ ✅] I have checked that there is no similar issue in the repo (**required**)
- [ ✅] I have provided a minimal working example to reproduce the bug (**required**)

### PR DESCRIPTION
**Pull request by Estalisium**
Update CVRPTW generator scaling to properly scale locations

## Description

This pull request modifies the scaling mechanism of the CVRPTW generator. 

## Motivation and Context

The previous implementation applied a uniform scaling factor (max_time) to both spatial (location) and temporal (time window) coordinates, which did not accurately reflect the distinct nature of these dimensions. The updated approach introduces separate scaling factors for locations and time windows.

 `close #258 ` 

- [✅ ] I have raised an issue to propose this change ([required](https://github.com/ai4co/rl4co/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Remove all that do not apply:

- [✅ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)
- [ ] Example (update in the folder of examples)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Estalisium** [comment]
Thank you for the clarification regarding the scaling approach. Sorry for the unwarranted update.
However, I've noticed that the current implementation of the render function sets fixed axis limits using ax.set_xlim(-0.05, 1.05) and ax.set_ylim(-0.05, 1.05). This approach can lead to display issues, especially when the plotted data falls outside these predefined limits even when it is scaled. This is why I assumed the scaling wasn't correct.
